aws_crt_common_ws/aws-crt-common/src/lib.rs

```diff
@@ -0,0 +1,596 @@
+/*
```

Comment:
probably makes sense to split this into 3 modules for errors, strings, and allocators